_layouts/default.html

```diff
@@ -3,22 +3,26 @@
   <head>
     <meta charset="utf-8">
     <title>Mocha - the fun, simple, flexible JavaScript test framework</title>
-    <link rel="stylesheet" href="normalize.min.css">
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="assets/normalize.min.css">
+    <link rel="stylesheet" href="assets/style.css">
     <link rel="shortcut icon" href="favicon.ico">
     <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.0/jquery.min.js"></script>
-    <script>window.jQuery || document.write('<script src="jquery.js"><\/script>')</script>
```

Comment:
I've removed jquery.js from the repo just to simplify things... lemme know if you want it back.